packages/smart-contracts/src/contracts/ERC20FeeProxy.sol

```diff
@@ -17,12 +17,16 @@ contract ERC20FeeProxy {
   );
 
   // Fallback function returns funds to the sender
-  function() external payable {
+  fallback() external payable {
     revert("not payable fallback");
   }
```

Comment:
Do you think we still need a fallback ? My understanding is that since Solidity 0.6 if `receive()` is implemented and we try to call a function that does not exist, it will fail, hence I don't see the need for a "manual" revert.

Cf. here: https://blog.soliditylang.org/2020/03/26/fallback-receive-split/

Reply:
(I won't replicate the comment to all the other fallback implementations in other contracts FYI so if you agree, think about those)

Reply:
Good point! Thanks.